### PR TITLE
💄  Add CSS and fix fenced blocks

### PR DESCRIPTION
**Pull request by bradystroud**
This issue fixed the broken builds https://github.com/gatsbyjs/gatsby/issues/22580

### CODE REVIEW

File: src/cms/previewTemplate.js [hunk 1]
```diff
@@ -27,8 +37,8 @@ var PreviewTemplate = ({ entry }) => {
   return (
     <body>
       <main>
-        <article className="rule-content">
-          <h1>{title}</h1>
+        <article className="rule-content editor-preview">
+          <h1 style={{ fontSize: '2.25rem' }}>{title}</h1>
```
Comment (pierssinclairssw) [inline src/cms/previewTemplate.js:41]: Should we be hardcoding the fontsize here or should it be css?

Comment (bradystroud) [reply src/cms/previewTemplate.js:41]: It should be CSS - I'll fix it 👍🏻

**Review by pierssinclairssw — APPROVED** [review]
Looks good.

**Review by pierssinclairssw — APPROVED** [review]
looks good

**Pull request merged**